test_rclcpp/test/test_multithreaded.cpp

```diff
@@ -204,13 +204,13 @@ TEST(CLASSNAME(test_multithreaded, RMW_IMPLEMENTATION), multi_consumer_clients)
       results.push_back(pair.first->async_send_request(pair.second));
     }
     // Wait on each future
-    for (uint32_t i = 0; i < results.size(); i++) {
+    for (uint32_t i = 0; i < results.size(); ++i) {
```

Comment:
I don't think so, because it is a `<` comparison it's fine for `i` to be a different size from `results.size()` since they're both unsigned. That's my guess at least. Anyways `i` needs to be `uint32_t` so it can be assigned into the message field below. Once we sort out the farm I'll rerun CI on this branch to see for sure.